persistence/blockstore/block_store.go

```diff
@@ -0,0 +1,65 @@
+package blockstore
+
+//go:generate mockgen -source=$GOFILE -destination=../types/mocks/blockstore/block_store_mock.go github.com/pokt-network/pocket/persistence/types BlockStore
+
+import (
+	"fmt"
+
+	"github.com/pokt-network/pocket/persistence/kvstore"
+)
+
+type BlockStore interface {
```

Comment:
Please add a small comment on why how this is used. E.g. "BlockStore is a key-value store mapping block heights to serialized block structures"